kernel/src/expressions/literal_expression_transform.rs

```diff
@@ -191,11 +191,13 @@ mod tests {
 
     use paste::paste;
 
+    use Expression as Expr;
```

Comment:
I think @scovich was aiming for "Use `Expr` as a shortcut in tests, but prod code should use `Expression` because it's more clear". So maybe don't do this :)

Reply:
yea shoot sorry actually not sure how this ended up in my diff - reverting